src/briefcase/platforms/windows/__init__.py

```diff
@@ -364,10 +371,21 @@ def package_app(
         except subprocess.CalledProcessError as e:
             raise BriefcaseCommandError(f"Unable to link app {app.app_name}.") from e
 
-        if sign_app:
-            self.logger.info("Signing MSI...", prefix=app.app_name)
-            self.sign_file(
-                app=app,
-                filepath=self.distribution_path(app),
-                **sign_options,
+    def _package_zip(self, app):
+        """Package the app as simple zip file."""
+
+        self.logger.info("Building zip file...", prefix=app.app_name)
+        with self.input.wait_bar("Packing..."):
+            zip_root_path = self.bundle_path(app) / f"{app.formal_name}-{app.version}"
+            self.tools.shutil.copytree(
+                self.bundle_path(app) / self.packaging_root,
+                zip_root_path,
+                dirs_exist_ok=True,
+            )
```

Comment:
This works, but it (a) results in a copy of the entire project, which might not be small, and (b) could be prone to stale files on the second build.

At the very least, the copied folder should be a one-use temporary folder, or deleted after use; however a better approach may be to *not* use `shutil`, and use zip file directly - that gives you much better control over the structure of the final zip file. For example, see [this StackOverflow](https://stackoverflow.com/a/32640292)  answer.

Reply:
I have considered some options. The 'problem' here is that we want to put the files in a root folder _that does not already exist_ before zipping. This is different to the situation described in the StackOverflow question/answer (that I also found). My impression was that neither `shutil.make_archive` nor `ZipFile` allows the creation/building of a root folder 'on the fly' while packing, so I was thinking about either to temporarily rename the `src` folder to `MyApp-1.2.3` (what I found 'hackish') or to make a plain copy and delete it afterwards (what I'm doing now).

OK, after reading this [StackOverflow answer](https://stackoverflow.com/questions/458436/adding-folders-to-a-zip-file-using-python), I see that it is possible.